tools/csv-generator/csv-generator.go

```diff
@@ -40,6 +40,13 @@ var (
 	uploadServerImage   = flag.String("uploadserver-image", "", "")
 	ovirtPopulatorImage = flag.String("ovirt-populator-image", "", "")
 	dumpCRDs            = flag.Bool("dump-crds", false, "optional - dumps cdi-operator related crd manifests to stdout")
+	dumpNetworkPolicies = flag.Bool("dump-network-policies", false, "optional - dumps cdi-operator related network-policy manifests to stdout")
+	dnsNamespace        = flag.String("dns-namespace", "", "optional - DNS namespace for the DNS network policy. Only used in conjunction with the dump-network-policies option")
+	dnsLabelKey         = flag.String("dns-pod-selector-label", "", "optional - DNS pod selector label key for the DNS network policy. Only used in conjunction with the dump-network-policies option")
+	dnsLabelValue       = flag.String("dns-pod-selector-value", "", "optional - DNS pod selector label value for the DNS network policy. Only used in conjunction with the dump-network-policies option")
+	apiNamespace        = flag.String("api-namespace", "", "optional - kube-apiserver namespace for the api network policy. Only used in conjunction with the dump-network-policies option")
+	apiLabelKey         = flag.String("api-pod-selector-label", "", "optional - kube-apiserver pod selector label key for the api network policy. Only used in conjunction with the dump-network-policies option")
+	apiLabelValue       = flag.String("api-pod-selector-value", "", "optional - kube-apiserver pod selector label value for the api network policy. Only used in conjunction with the dump-network-policies option")
```

Comment:
Maybe you can defuault these to `kube-system`, `component`, `kube-apiserver`,	`kube-system`, `k8s-app`, `kube-dns`